server/config/config.go

```diff
@@ -109,7 +109,7 @@ var (
 
 	evmTracers = []string{"json", "markdown", "struct", "access_list"}
 
-	blockExecutors = []string{BlockExecutorSequential, BlockExecutorBlockSTM}
+	blockExecutors = []string{BlockExecutorSequential, BlockExecutorBlockSTM, ""}
```

Comment:
i think it defaults to `"sequential"`.

Reply:
seems easy to forgot to add this config when upgrade, and app treat empty same as [sequential](https://github.com/crypto-org-chain/ethermint/blob/0e0baa4ea54079296cebf233fe7510700f981830/app/app.go#L346) while validation doesn't.

Reply:
is `sequential` as default fine?

Reply:
yes